### ISSUE TITLE
Add config option for PHP Beautifier filters

### ISSUE THREAD

**Issue by joeguilmette** [Glavin001/atom-beautify#107]
So, my journey began when I wanted to easily beautify a particularly nasty PHP array. "I'll just use this neat looking package, `Atom Beautify`!" I said to myself. Turns out it requires `PHP Beautifier`. No problem, I'll just install it. Well, the package on the Atom package manager 404'd, so I went ahead to manually install it. And then I needed Pear. And then I needed to add it's include path to php.ini, but actually there is no php.ini so I needed to copy it over from wherever it was hiding (don't ask because I really have no idea where it was). Then I was finally able to add the include_path for Pear into php.ini, but only after googling around to figure out how nano works so as to avoid borking things further. I'm not willing to admit how many hours it took to get there.

Imagine my despair when `php_beautifier`, used through `Atom Beautify`, simply compressed the entire array into a single line. Not ideal!

Upon further research, I have found that there are filters out there that will help, but...

**my real question is,**

**_tl;dr:_ How do I use `php_beautify` filters with `Atom Beautify`?**
